tests/models/test_hparams.py

```diff
@@ -521,7 +529,7 @@ def test_hparams_save_yaml(tmpdir):
     assert load_hparams_from_yaml(path_yaml) == hparams
 
 
-class NoArgsSubClassEvalModel(EvalModelTemplate):
+class NoArgsSubClassBoringModel(CustomBoringModel):
```

Comment:
Why do we need this class instead of just BoringModel?

Reply:
true. no need for this. will update.

Reply:
I guess it's required to check the args for a class that is inherited from a model that requires args.

Reply:
at least, we don't need to re-define forward, loss, training and test step, ...

Reply:
@rohitgr7

Reply:
done.